pkg/cluster/config.go

```diff
@@ -340,3 +353,16 @@ func IsCloudControllerOwnedByCCM(oclient configclient.Interface) (bool, error) {
 
 	return ownedByCCM, nil
 }
+
+// getProxyVars returns a map of the proxy variables and values from the WMCO container's environment. The presence of
+// any implies a proxy is enabled, since OLM would inject into the operator spec. Returns an empty map otherwise.
+func getProxyVars() map[string]string {
+	proxyVarsMap := make(map[string]string, len(acceptableProxyVars))
```

Comment:
Please don't call out the data structure used in the variable now. That is an anti-pattern in go. `proxyVars` will suffice.